masterlist.yaml

```
@@ -2679,6 +2679,17 @@ plugins:
         util: '[FO4Edit v4.0.3h (Hotfix 1)](https://www.nexusmods.com/fallout4/mods/2737)'
         itm: 3
         udr: 9
+  
+  - name: 'Vault81Etiquette.esp'
```

There are trailing spaces here:

```suggestion
        udr: 9

  - name: 'Vault81Etiquette.esp'
```